zetasql/fuzzing/protobuf/zetasql_expression_extractor_test.cc

```diff
@@ -0,0 +1,267 @@
+//
+// Copyright 2020 Google LLC.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+#include <memory>
+#include <string>
+
+#include "zetasql/fuzzing/protobuf/zetasql_expression_grammar.pb.h"
+#include "zetasql/fuzzing/protobuf/zetasql_expression_extractor.h"
+#include "gtest/gtest.h"
+
+using zetasql_expression_grammar::Expression;
+using zetasql_expression_grammar::LiteralExpr;
+using zetasql_expression_grammar::IntegerLiteral;
+using zetasql_expression_grammar::NumericLiteral;
+using zetasql_expression_grammar::CompoundExpr;
+using zetasql_expression_grammar::BinaryOperation;
+using parameter_grammar::Whitespace;
+using zetasql_fuzzer::ProtoExprExtractor;
+
+namespace zetasql_fuzzer {
```

Comment:
nit: I'd move this above the using statements, then add another unnamed namespace just to enforce internal linkage.

namespace zetasql_fuzzer {
namespace {

using ...

...

Reply:
FRIEND_TEST requires Test class in the same scope as the target class.